src/neptune_airflow/impl/__init__.py

```diff
@@ -65,14 +65,41 @@ def getinstance(*args, **kwargs):
 
 @singleton
 class NeptuneLogger:
+    """Creates a Neptune logger instance for tracking metadata during a DAG run.
+
+    Args:
+        api_token: User's Neptune API token.
+            If None, the value of the `NEPTUNE_API_TOKEN` Airflow Variable is used (recommended).
+            For help, see https://docs.neptune.ai/setup/setting_api_token/
```

Comment:
@AleksanderWWW As I recall, you mentioned that in this case, Airflow Variables are the recommended way of setting Neptune credentials. I think it's OK to mention just that option here, but in the API reference docs we can list any alternatives.

Does it also work normally if the user has `NEPTUNE_API_TOKEN` and `NEPTUNE_PROJECT` set as regular environment variables?

Reply:
Agreed. Airflow Variables will have precedence over env variables. The hierarchy is as follows:
- data passed explicitly by the user (though not recommended especially in distributed systems as they'll have to be passed again if another task is on a different machine)
- Airflow Variables (recommended - you set them once and they work for all tasks)
- Env variables ( would need to be careful to set them on all machines)

Reply:
Got it! Thanks for the extra context.